examples/tests/cluster-upgrade/step2/index.ts

```diff
@@ -0,0 +1,48 @@
+
+import * as pulumi from "@pulumi/pulumi";
+import * as awsx from "@pulumi/awsx";
+import * as eks from "@pulumi/eks";
+import * as iam from "./iam";
+
+const config = new pulumi.Config();
+const clusterVersion = config.require("upgradeVersion");
```

Comment:
I wish ProgramTest steps could just indicate different config values to avoid dup'ing source.